src/lib.rs

```diff
@@ -820,8 +829,8 @@ pub struct Library {
     pub name: String,
     /// From where the library settings have been retrieved
     pub source: Source,
-    /// libraries the linker should link on
-    pub libs: Vec<String>,
+    /// libraries the linker should link on, true if a static lib is available
+    pub libs: Vec<(String, bool)>,
```

Comment:
Maybe it would be clearer to create an `InternalLib` struct instead of using a tuple here?